lms/templates/header/user_dropdown.html

```diff
@@ -1,16 +1,18 @@
 ## mako
 <%page expression_filter="h"/>
-<%namespace name='static' file='static_content.html'/>
+<%namespace name='static' file='../static_content.html'/>
```

Comment:
Was this change intentional? Why do we now need to pull from up a directory?

Reply:
There are unintentional thank you for the catch!